app.py

```diff
@@ -0,0 +1,21 @@
+from flask import (
+    jsonify,
+    render_template,
+    Flask,
+)
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
```

Comment:
So this bit pulls the template you have for index in our template folder?

Reply:
Yup!